app/views/quick_search/pages/home.html.erb

```diff
@@ -0,0 +1,50 @@
+<% title %>
+<%= render partial: '/layouts/quick_search/page_title', locals: { page_title: @page_title } %>
+<%= render partial: 'layouts/quick_search/search_form' %>
+
+<div class='row search-helper-text'>
+  <div class='col-12 text-center'>
+    <%= image_tag 'handle.svg', class: 'img-fluid' %>
+  </div>
+  <div class='col-12 text-center'>
+      One search to see what's available in all these sources for your search topic.
+  </div>
+</div>
+<div class='row homepage-panels'>
+  <div class='col'>
+    <div class='card'>
+      <%= image_tag 'catalog.jpg', class: 'card-img-top' %>
```

Comment:
Do these images need alt text or `aria-hidden`?

Reply:
This is implemented the same way as @jvine has specified in SearchWorks. By default they are getting an `alt=`